lib/bloc/user_profile/security_model.dart

```diff
@@ -1,28 +1,34 @@
-class SecurityModel {
+enum BackupKeyType { NONE, PIN, PHRASE }
 
+class SecurityModel {
   static List<int> lockIntervals = List.unmodifiable([0, 30, 120, 300, 600, 1800, 3600]);
   static const int _defaultLockInterval = 120;
 
-  final bool requiresPin;  
-  final bool secureBackupWithPin;
+  final bool requiresPin;
+  final BackupKeyType backupKeyType;
   final int automaticallyLockInterval;
 
-  SecurityModel({this.requiresPin, this.secureBackupWithPin, this.automaticallyLockInterval});
+  SecurityModel({this.requiresPin, this.backupKeyType, this.automaticallyLockInterval});
```

Comment:
Better makes this private constructor.

Reply:
[Done](https://github.com/breez/breezmobile/pull/173/commits/1aac4e35785b3b6da90e92c822696485bf9953e8)